packages/python-pulp-rpm/python-pulp-rpm.spec

```diff
@@ -34,8 +34,8 @@ Requires:       python%{python3_pkgversion}-jsonschema >= 3.0
 Requires:       python%{python3_pkgversion}-libcomps >= 0.1.15
 Conflicts:      python%{python3_pkgversion}-libcomps >= 0.2
 Requires:       python%{python3_pkgversion}-productmd >= 1.33
-Requires:       python%{python3_pkgversion}-pulpcore < 3.16
-Requires:       python%{python3_pkgversion}-pulpcore >= 3.14.2
+Requires:       python%{python3_pkgversion}-pulpcore < 3.15
```

Comment:
I wonder if that's intentional upstream?
(But it correctly reflects the change in requirements.txt)

Reply:
i kinda assumed it was due to all the 3.15 changes, but i can inquire to confirm

Reply:
```
  dralley | it was intentional   
```